### ISSUE TITLE
Import URL list

### ISSUE THREAD

**Issue by klorydryk** [wallabag/wallabag#3757]
Hello,

After having lost my Wallabag server, I've exported the wallabag android app DB. I tried to import it into the Wallabag server, but I have now issues: https://github.com/wallabag/android-app/issues/738

One solution could be to allow user to import a simple list of url in any format: CSV, 1 url/line... and use it as new articles. Something we could name a "raw format".

Thanks !

**Comment by Trevo525** [comment]
I am coming from Raindrop and wanting to import. I would love a way to do that with the .csv that it can export, but if it has to be a text file with a URL on each line I would be happy with that as well.

**Comment by nicosomb** [comment]
@Trevo525 could you send us an example of Raindrop export please?

**Comment by nicosomb** [comment]
If we want to keep tags, csv and html are the good choice. 
If we want to faster, txt seems to be easy to parse 😅

**Comment by Trevo525** [comment]
I of course would like to keep my tags, but, if we were able to import a text file with a list of URLs that seems like something that would be generic enough that you could probably import from pretty much any service without more development in the future..

**Comment by nicosomb** [comment]
I read the first post of @klorydryk again. 

> One solution could be to allow user to import a simple list of url in any format: CSV, 1 url/line... and use it as new articles. Something we could name a "raw format".

The initial goal of this feature request is to import an easy file (I'll update the title). 

@Trevo525 maybe you can open a new issue if you want a "complex" Raindrop export.

**Comment by Trevo525** [comment]
Honestly, I'd be super happy with just the feature of this request.

**Comment by emelinea-hina** [comment]
Agreed, there needs to be a way to import simple data formats. Today i tried to extract my favorites out of an old akregator database, and to import that into wallabag, i'm looking into imitating the export file format of other applications, which aren't terribly well documented. What should have been a simple string interpolation is turning into a whole endeavor.

Edit: I decided to go with imitating Pocket's import, [it's just a few columns](https://github.com/wallabag/wallabag/blob/17398e68e5253aa5eb9ed9996e941b8ff18790b0/src/Import/PocketCsvImport.php#L63). To summarize for anyone looking to do the same thing that I did:

* it's CSV decoded using PHP's fgetcsv
* any header (names for the columns) is ignored
* rows are in that order: title, url, timestamp, tags, archival status
* the tags can be separated with "|" or "," (haven't tested that one but if i read the source correctly it's this)
* for archival status write "archive" if it's archived, nothing (empty string) if it's not
* the timestamp is expected to be a UNIX timestamp
